app/controllers/sessions_controller.rb

```diff
@@ -2,54 +2,84 @@ class SessionsController < ApplicationController
   def new
   end
 
+  def shop_owner_login
```

Comment:
Yes, because I only needed it to serve as the controller method to render the form. I didn't need to pass any information through it... just like the `new` method above it.